hello-world-python/enclave/build.gradle

```diff
@@ -0,0 +1,40 @@
+plugins {
+    id 'com.r3.conclave.enclave'
+}
+
+dependencies {
+    testImplementation "org.junit.jupiter:junit-jupiter:5.6.0"
+}
+
+conclave {
+    productID = 1
+    revocationLevel = 0
+
+    simulation {
+        signingType = privateKey
+        signingKey = file("../signing/sample_private_key.pem")
+        supportLanguages = "python"
```

Comment:
Line 16 should be moved to line 12.

Reply:
done. apologies I had committed but not pushed this one.